public/locales/en/profile.json

```diff
@@ -0,0 +1,24 @@
+{
+    "10ticketsRemaining": "10 Tickets Remaining",
+    "3cardsAdded": "3 Cards Added",
```

Comment:
Same here with en keys
[How To Pass a Variable to Your Translation String With i18next](https://betterprogramming.pub/how-to-pass-a-variable-to-your-translation-string-with-i18next-b07614c33514)